Week01/Problem03/asedler_03.py

```diff
@@ -0,0 +1,20 @@
+"""
+This script implements a solution to Project Euler Problem 03
+by finding the largest prime factor of 600851475143.
+
+@author Andrew Sedler
+"""
+
+# Assign the number to be factored
+x = 600851475143
+
+# Start with the smallest prime
+i = 2
+
+while i*i <= x:
```

Comment:
What's the largest number which could possibly be a prime factor of `x`? How could you minimize the amount of work you're doing by starting with checking that number?